### ISSUE TITLE
Can't deposit ironfish

### ISSUE THREAD

**Issue by Amiron1** [iron-fish/ironfish#1349]
### What happened?

I was trying to deposit to the bank, byt something went wrong. Everything's working too slow, transactions take ages.

### Version

ubuntu 20.04

### Debug output

```shell
Database in use, skipping output that requires database.
Stop the node and run the debug command again to show full output.

Iron Fish version       0.1.32 @ src
Iron Fish library       0.0.10 @ src
Operating system        Linux x64
CPU model(s)            Intel Core Processor (Haswell, no TSX, IBRS)
CPU threads             6
RAM total               15.63 GiB
Node version            v16.15.0
ironfish in PATH        true
Telemetry enabled       true
Done in 6.28s.
```

### Relevant log output

```shell
You are about to send $IRON 0.10000000 ($ORE 10,000,000) plus a transaction fee of $IRON 0.00000001 ($ORE 1) to the Iron Fish deposit account.
Your remaining balance after this transaction will be $IRON 0.10897054 ($ORE 10,897,054).
The memo will contain the graffiti "Andreynode".

* This action is NOT reversible *

Do you confirm (Y/N)?: y
Creating the transaction: [████████████████████████████████████████] 100% | ETA: 0s
An error occurred while sending the transaction.
 ›   Error: Request failed (400) insufficient-balance: Please wait a few seconds for your balance to update and try again
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
root@rs003897:~# yarn start debug
yarn run v1.22.18
error Couldn't find a package.json file in "/root"
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

### Graffiti

Andreynode

**Comment by roadswithoutend** [comment]
I have the same issue
The balance is: $IRON 68.39998985 ($ORE 6,839,998,985)
Amount available to spend: $IRON 28.39998832 ($ORE 2,839,998,832)
Done in 27.53s.

You are about to send $IRON 0.10000000 ($ORE 10,000,000) plus a transaction fee of $IRON 0.00000001 ($ORE 1) to the Iron Fish deposit account.
Your remaining balance after this transaction will be $IRON 28.29998831 ($ORE 2,829,998,831).
The memo will contain the graffiti "roadswithoutend".

This action is NOT reversible *
Do you confirm (Y/N)?: y
Creating the transaction: [████████████████████████████████████████] 100% | ETA: 0s
An error occurred while sending the transaction.
› Error: Request failed (400) error: Insufficient funds
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

**Comment by lwisne** [comment]
We believe this is fixed now. Comment here if you're still experiencing this issue!
